examples/common/utils.py

```diff
@@ -105,6 +105,8 @@ async def store_secret_array(
         secret_array (np.ndarray): Secret array.
         name (str): Secrets name.
         nada_type (Any): Nada type.
+        permissions: nillion.Permissions: Optional Permissions.
```

Comment:
nit: docstring formatting is a bit different here than in the rest of the codebase

Reply:
updated to match:

```
permissions (nillion.Permissions): Optional Permissions.
```